urdf/ergoCub/conf/wrappers/motorControl/right_arm-mc_remapper.xml

```diff
@@ -4,9 +4,9 @@
 <device xmlns:xi="http://www.w3.org/2001/XInclude" name="right_arm-mc_remapper" type="controlboardremapper">
     <paramlist name="networks">
         <elem name="right_arm_joints1">(  0  6  0 6 )</elem>
-        <elem name="right_arm_joints2">(  7  12 0 5 )</elem>
+        <elem name="right_arm_joints2">(  7  18 0 11 )</elem>
     </paramlist>
-    <param name="joints"> 13 </param>
+    <param name="joints"> 19 </param>
```

Comment:
Please undo also this change, I think you had to change it because you removed `reserved` in the ini, but the joints in the arms are 13:

https://github.com/robotology/robots-configuration/blob/a70237cd9f7805134cd7735e97ebb20e6ae471ba/ergoCubSN000/wrappers/motorControl/left_arm-mc_remapper.xml#L6